source/rust.txt

```diff
@@ -30,6 +30,8 @@ or set up a runnable project with examples from our Usage Guide.
 
 - `Source Code <https://github.com/mongodb/mongo-rust-driver/>`__
 
+- `The Rust Driver Manual <https://mongodb.github.io/mongo-rust-driver/manual/>`__
```

Comment:
Suggestion: I think this link should be brought up to either the first or second bullet. I feel that it's a bit hidden as the last item and can easily be skimmed over.